pkg/server/controller/op.go

```diff
@@ -34,7 +34,7 @@ func (c *OpController) HandleLogin(ctx *gin.Context) (interface{}, error) {
 
 	var res plugin.Response
 	token := content.Metas["token"]
-	if c.tokens[content.User] == token {
+	if (len(content.User) > 0 && len(token) > 0) && c.tokens[content.User] == token {
```

Comment:
Can we give a more accurate error message for this like `user or meta token can not be empty`?

Reply:
Done. Is this ok?